daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -1074,13 +1074,15 @@ private[archive] class DecodeV1(minor: LV.Minor) extends Decode.OfPackage[PLF.Pa
               "Update.Exercise.choice.choice"
             ),
             cidE = decodeExpr(exercise.getCid, definition),
-            actorsE =
+            actorsE = if (versionIsOlderThan(LV.Features.noExerciseActor)) {
```

Comment:
Do we still support decoding DAML-LF 1.5 or older? If we do, wouldn't that mean that we still have to deal with both cases, `actors` present or absent, in the interpreter and this change would not really buy us anything?

Reply:
DAML-LF 1.5 or latter are deprecated since #5321 that is before the release of SDK 1.0.0. 
We will drop support of  DAML-LF 1.5 or older in the future however for the sake of consistency I prefer not to break reading older version.